src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -68,6 +68,19 @@
  * &lt;module name=&quot;FinalParameters&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   public MyClass(final int num) { // OK
+ *   }
+ *   public MyClass(long l) { // violation, l should be final
```

Comment:
`public MyClass(long value)`
no single char symbols.

please use this class in all examples, so only comment "//OK" and "//violation" are changing between examples

Reply:
done